src/test/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -71,11 +71,8 @@ class BasicWorkflow extends FunSpec with Matchers {
         val javaSearchSymbol = project.rpcPublicSymbolSearch(List("java", "io", "File"), 2)
         javaSearchSymbol match {
           case SymbolSearchResults(List(
-            TypeSearchResult("java.awt.image.ImageObserver", "ImageObserver", 'class,
```

Comment:
@jschlather Have we broken the test here - looking up from java.io.File looks like it used to return sensible results and now returns nothing?